mmv1/templates/tgc/resource_converters.go.erb

```diff
@@ -49,26 +49,26 @@ func ResourceConverters() map[string][]cai.ResourceConverter {
 		"google_compute_health_check":                     {compute.ResourceConverterComputeHealthCheck()},
 		"google_compute_instance":                         {compute.ResourceConverterComputeInstance()},
 		"google_compute_instance_group":                   {resourceConverterComputeInstanceGroup()},
-                "google_compute_network":                          {compute.ResourceConverterComputeNetwork()},
```

Comment:
Please remove all whitespace changes in this file

Reply:
sorry i did but every time that i solve a conflit the spaces back